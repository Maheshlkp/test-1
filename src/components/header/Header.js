import React from "react";


export const Header = () => {
  return (
    <header>
      <h1 className="header">Calculator</h1>
    </header>
  );
};
