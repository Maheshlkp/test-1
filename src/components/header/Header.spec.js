import React from 'react';
import {shallow} from 'enzyme';
import {Header} from './Header';

describe('Test Header', () => {

    it('should render correctly', () => {
        const wrapper = shallow(<Header />);
        expect(wrapper.getElements()).toMatchSnapshot();
    })
});