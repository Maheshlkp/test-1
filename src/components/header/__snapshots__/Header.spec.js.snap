// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Header should render correctly 1`] = `
Array [
  <header>
    <h1
      className="header"
    >
      Calculator
    </h1>
  </header>,
]
`;
